routes/extra_data_utils.go

```diff
@@ -0,0 +1,38 @@
+package routes
+
+import (
+	"encoding/hex"
+	"github.com/deso-protocol/core/lib"
+	"strconv"
+)
+
+type DecoderFunc func([]byte) string
+
+type ExtraDataDecoder struct {
+	Key     string
+	Decoder DecoderFunc
+}
+
+// GetExtraDataDecoderFunc Values in ExtraData field
+// in transaction may have special encoding. In such cases
+// we'll need specialized decoders too
+func GetExtraDataDecoderFunc(key string) DecoderFunc {
+	var decoders = []ExtraDataDecoder{
+		{
+			lib.DiamondLevelKey,
+			ByteArrTo64BitInt,
+		},
+	}
```

Comment:
Maybe we create a map outside of this function, called ExtraDataKeyToDecoder map so we don't have to iterate over all the decoders below.